staging/src/k8s.io/component-base/metrics/testutil/testutil.go

```diff
@@ -68,3 +70,25 @@ func NewFakeKubeRegistry(ver string) metrics.KubeRegistry {
 
 	return metrics.NewKubeRegistry()
 }
+
+// GetMetricLabelValue returns string that consists of the value of labelName in Metric
+func GetMetricLabelValue(element interface{}, labelName ...string) string {
+	el := element.(*model.Sample)
```

Comment:
Let's reuse the strategy that we decide in [the other PR](https://github.com/kubernetes/kubernetes/pull/90397#discussion_r414267212), rather than do this. I'm not sure how using an empty interface is going to help here.